src/Castle.Windsor/Compatibility/AppContext.cs

```diff
@@ -17,7 +17,10 @@
 #if DOTNET45
 namespace Castle.Core.Internal
 {
-    internal static class AppContext
+	// This should be changed to expose externals similar to how core does it. Rather then copy this down to tests we should 
+	// be using the InternalsVisibleTo attribute. Will raise this on the PR in GitHub. Making it public for now to get
+	// the project to compile. If you are reviewing this, please flag it.
+    public static class AppContext
```

Comment:
I think I just reviewed @alinapopa get rid of the duplicate class in the other open PR.